src/components/EndorsementList.astro

```diff
@@ -0,0 +1,69 @@
+---
+// © 2024 Vlad-Stefan Harbuz <vlad@vladh.net>
+// SPDX-License-Identifier: Apache-2.0
+
+import Endorsement from "./Endorsement.astro";
+
+interface Props {
+  full?: boolean;
+}
+
+const { full } = Astro.props;
+---
+
+<div class="endorsement-list" data-full={full}>
```

Comment:
I guess we should sort by impact, eh? Or maybe next week we go with sections?

1. OSI at the top
2. Other foundations/institutions, sorted by date (desc?)
3. Individuals, sorted by date (desc?)

Reply:
This is fine for today. 👍